src/main/java/ch/njol/skript/expressions/ExprTarget.java

```diff
@@ -108,39 +140,27 @@ public boolean setTime(int time) {
 	}
 
 	@Override
-	@Nullable
-	public Class<?>[] acceptChange(ChangeMode mode) {
-		if (mode == ChangeMode.SET || mode == ChangeMode.DELETE)
-			return CollectionUtils.array(LivingEntity.class);
-		return super.acceptChange(mode);
+	public Class<? extends Entity> getReturnType() {
+		return type != null ? type.getType() : Entity.class;
 	}
 
 	@Override
-	public void change(Event e, @Nullable Object[] delta, ChangeMode mode) {
-		if (mode == ChangeMode.SET || mode == ChangeMode.DELETE) {
-			LivingEntity target = delta == null ? null : (LivingEntity) delta[0];
-			for (LivingEntity entity : getExpr().getArray(e)) {
-				if (getTime() >= 0 && e instanceof EntityTargetEvent && entity.equals(((EntityTargetEvent) e).getEntity()) && !Delay.isDelayed(e)) {
-					((EntityTargetEvent) e).setTarget(target);
-				} else if (entity instanceof Mob) {
-					((Mob) entity).setTarget(target);
-				}
-			}
-			return;
-		}
-		super.change(e, delta, mode);
+	public String toString(@Nullable Event e, boolean debug) {
+		if (e == null)
+			return "the target" + (type == null ? "" : "ed " + type) + (getExpr().isDefault() ? "" : " of " + getExpr().toString(e, debug));
+		return Classes.getDebugMessage(getAll(e)); // TODO for players it will return null (need to getTarget)
 	}
 
 	/**
 	 * Gets an entity's target.
-	 * 
+	 *
 	 * @param entity The entity to get the target of
-	 * @param type Can be null for any entity
+	 * @param type   Can be null for any entity
```

Comment:
Mistake tab/spaces insertion?
```suggestion
	 * @param type The exact EntityData to find. Can be null for any entity.
```

Reply:
My IDE suggested that to align the description together